test/utils/utils.go

```
@@ -296,7 +326,9 @@ func GetProjectDir() (string, error) {
 	if err != nil {
 		return wd, err
 	}
+	// Remove common test directory suffixes to get to project root
 	wd = strings.Replace(wd, "/test/e2e", "", -1)
```

golangci-lint (2.12.2)

[error] QF1004: could use strings.ReplaceAll instead

(staticcheck)